ajax/libs/proteic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "proteic",
+  "filename": "proteic.js",
+  "version": "0.0.27",
+  "description": "Streaming and static data visualization for the modern web",
+  "homepage": "http://proteic.js.org/",
+  "keywords": [
+    "d3",
+    "chart",
+    "data visualization"
+  ],
+  "author": {
+    "name": "Treelogic S.L.",
+    "url": "http://www.treelogic.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/proteus-h2020/proteic.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/itsjavi/fontawesome-iconpicker.git",
```

Comment:
This link can't get all the version of `proteic` lib.
Would you modify it? please.
Thank you.